docs/framework/debug-trace-profile/tracing-and-instrumenting-applications.md

```diff
@@ -128,7 +128,7 @@ System.Diagnostics.Debug.WriteLine("Hello World!");
  You can produce custom results by implementing your own listener. A custom trace listener might, for example, display the messages to a message box, or connect to a database to add messages to a table. All custom listeners should support the six methods mentioned above. For more information on creating developer-defined listeners, see <xref:System.Diagnostics.TraceListener> in the .NET Framework reference.  
   
 > [!NOTE]
->  In Visual Basic, the **Debug.Write**, **Debug.WriteIf**, **Debug.WriteLine**, and **Debug.WriteLineIf** methods have replaced the **Debug.Print** method that was available in earlier versions of Visual Basic.  
+>  In Visual Basic 2005, the **Debug.Write**, **Debug.WriteIf**, **Debug.WriteLine**, and **Debug.WriteLineIf** methods have replaced the **Debug.Print** method that was available in earlier versions of Visual Basic.  
```

Comment:
This note applies to Visual Basic 2002 and 2003. And Debug.Print was reintroduced in .NET Framework 2.0. It's best to remove this note entirely.